config/Version.Public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=8.6
 
 // Public long version example: VERSION_LONG=1.2.0.0
-VERSION_LONG=8.6.0.1
+VERSION_LONG=8.6.0.3
```

Comment:
Based on the commit history, I assume there wasn't a PR for `8.6.0.2` wasn't made or merged?

Reply:
Correct – two betas close enough together that everything landed in this one.